Assignment/Homework_1 - HTML/Pokedex.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Pokedex</title>
+	<header>
+		<button type="button">Dropdown</button>
+		<a href="http://www.pokemon.com/us/pokedex/"><h1 id="title">Pokedex</h1></a>
+		<img src="assets/icons/pokeball_icon.png">
+	</header>
+</head>
+<body background="assets/backgrounds/background_home.png">
+	<div>
+		<input type="text" name="search" placeholder="Search Pokemon">
+	</div>
+
+	<div>
+		<span>
+			<img src="assets/images/pickahu.png">
+		</span>
+		<span id="area">
+			<section>
+				<h2>No. 001 Pikachu</h2>
+				<ul>
+					<li>Electric type</li>
+					<li>Gentle temperament</li>
+					<li>Grasslands</li>
+				</ul>
+			</section>
+		</span>
+	</div>
+	<div>
+		<article>
+			<p id="next">Pikachu can be seen napping in a bright sunlight. It has two red dots on its
+			cheeks. By practicing html and css Pikachu can learn confusion. Pikachu
+			likes to steal computer chargers from sleeping college students.</p>
+		</article>
```

Comment:
article used for semantic structure. putting 'article' in a 'div' defeats the purpose of using 'article'